batch/src/test/scala/uk/gov/ons/addressindex/models/HybridAddressSkinnyNisraEsDocumentSpec.scala

```diff
@@ -103,28 +103,19 @@ class HybridAddressSkinnyNisraEsDocumentSpec extends WordSpec with Matchers {
   val actualNagLpiLastUpdateDate = new java.sql.Date(format.parse("2012-04-24").getTime)
   val actualNagLpiEndDate = new java.sql.Date(format.parse("2018-01-11").getTime)
 
-  // NISRA example
-  val nisraOrganisation = "AN ORGANISATION"
-  val nisraSubBuildingName = "THE SUB BUILDING NAME"
-  val nisraBuildingName = "THE BUILDING NAME"
-  val nisraBuildingNumber = "1A"
-  val nisraThoroughfare = "THOROUGHFARE ROAD"
-  val nisraDependentThoroughfare = "OFF HERE"
-  val nisraAltThoroughfare = "AN ALTERNATIVE NAME"
-  val nisraLocality = "A LOCALITY XYZ"
-  val nisraTownland = "BIG TOWNLAND"
-  val nisraTown = "LITTLE TOWN"
-  val nisraPostCode = "AB1 7GH"
-  val nisraEasting = 379171.00F
-  val nisraNorthing = 412816.00F
+  // NISRA location (shared field to avoid test failure)
   val nisraLocation = Array(-2.3162985F, 4.00F)
-  val nisraUprn = 100010977866L
-  val nisraCreationDate = new java.sql.Date(format.parse("2012-04-23").getTime)
-  val nisraCommencementDate = new java.sql.Date(format.parse("2012-04-24").getTime)
-  val nisraArchivedDate = new java.sql.Date(format.parse("2018-01-11").getTime)
-  val nisraMixed = "An Organisation, The Sub Building Name, The Building Name, 1A Off Here, Thoroughfare Road, A Locality Xyz, Big Townland, Little Town, AB1 7GH"
-  val nisraAltMixed = "An Organisation, The Sub Building Name, The Building Name, 1A Off Here, An Alternative Name, A Locality Xyz, Big Townland, Little Town, AB1 7GH"
-  val nisraAll = "AN ORGANISATION THE SUB BUILDING NAME THE BUILDING NAME 1A OFF HERE THOROUGHFARE ROAD AN ALTERNATIVE NAME A LOCALITY XYZ BIG TOWNLAND LITTLE TOWN AB1 7GH"
+
```

Comment:
These vals appear to be constants which is fine but other expected vals in the class are not named as constants. We should maintain consistency.

Reply:
will change to lower case first letter to be consistent (and probably correct according to https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods )